components/console-backend-service/internal/domain/eventing/trigger.go

```diff
@@ -0,0 +1,119 @@
+package eventing
+
+import (
+	"context"
+	"github.com/golang/glog"
+
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/domain/eventing/listener"
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/domain/eventing/pretty"
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/domain/eventing/trigger"
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/gqlerror"
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/gqlschema"
+
+	"github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
+	"github.com/pkg/errors"
+)
+
+type triggerResolver struct {
+	service   trigger.Service
+	converter trigger.GQLConverter
+}
+
+func newTriggerResolver(svc trigger.Service, converter trigger.GQLConverter) *triggerResolver {
+	return &triggerResolver{
+		service:   svc,
+		converter: converter,
+	}
+}
+
+func (r *triggerResolver) TriggersQuery(ctx context.Context, namespace string, subscriber *gqlschema.SubscriberInput) ([]gqlschema.Trigger, error) {
+	triggers, err := r.service.List(namespace, subscriber)
+	if err != nil {
+		glog.Error(errors.Wrapf(err, "while finding %s in namespace `%s`", pretty.Triggers, namespace))
+		return nil, gqlerror.New(err, pretty.Triggers)
+	}
+
+	return r.converter.ToGQLs(triggers)
+}
+
+func (r *triggerResolver) CreateTrigger(ctx context.Context, trigger gqlschema.TriggerCreateInput, ownerRef []gqlschema.OwnerReference) (*gqlschema.Trigger, error) {
+	converted, err := r.converter.ToTrigger(trigger, ownerRef)
+	if err != nil {
+		glog.Error(errors.Wrapf(err, "while converting %s `%s`", pretty.Trigger, trigger.Name))
```

Comment:
glog.Error(errors.Wrapf(err, "while converting input `%s` to %s", trigger.Name, pretty.Trigger))